### ISSUE TITLE
Use pylint to lint check all examples

### ISSUE THREAD

**Issue by tannewt** [adafruit/Adafruit_Learning_System_Guides#2]
It enforces pep8: https://github.com/PyCQA/pycodestyle

**Comment by tannewt** [comment]
We do this in libraries already and should follow the example settings there: https://github.com/adafruit/cookiecutter-adafruit-circuitpython/blob/master/%7B%7B%20cookiecutter.library_name%20%7D%7D/.travis.yml#L30

**Comment by craigargh** [comment]
I'm going to work on:
- simple_strand_palettes
- uf2_flash_erasers
- ufo

**Comment by ladyada** [comment]
one possible route - have the pylint script live in the root of the repo, and run on any .py script it finds, UNLESS a .circuitpython.skip file exists in the same directory of the .py script

**Comment by dylwhich** [comment]
I'm working on adding a pylint script and a travis-ci config to run it

**Comment by craigargh** [comment]
@dylwhich Would you be able to add a pylint config file as well?

**Comment by ajdershem** [comment]
I am working on:

- 3D_Printed_NeoPixel_Gas_Mask

**Comment by craigargh** [comment]
I'm using the following to run pylint
```
pylint --disable=missing-docstring,invalid-name,bad-whitespace,duplicate-code,import-error,consider-using-enumerate,no-name-in-module,no-member,too-many-arguments,no-else-return */*.py
```

It might make sense to Flake8 instead of PyLint as it's less opinionated.

**Comment by kattni** [comment]
@craigargh `@staticmethod` is implemented in CircuitPython.

**Comment by dhalbert** [comment]
Fixed by #166. Thanks @craigargh !
